components/form/form.js

```diff
@@ -136,13 +140,27 @@ class Form extends FormMixin(LitElement) {
 		e.stopPropagation();
 		e.preventDefault();
 		const form = e.composedPath()[0];
-		this._nestedForms.set(e.target, form);
```

Comment:
Would we still need to use an array if the key was `form`, rather than `e.target`?

Reply:
It would definitely affect how the validation & submission happens, since `_findFormElements` uses the presence of an element in this DOM scope (`e.target` here) to add it to the list of things that needs validation/submitting.